### PR TITLE
Adds auto-scaling to LinearSVC and adds psudeo-Feature modifiers.log

### PR DESCRIPTION
**Pull request by halfak**

### CODE REVIEW

File: revscores/scorers/linear_svc.py [hunk 1]
```diff
@@ -74,7 +78,20 @@ def test(self, values_scores):
...
+        columns = zip(*values)
+        
+        stats = tuple((mean(c), stdev(c)) for c in columns)
         
+        return stats
+    
+    def _scale_and_center(self, values, stats):
+        
+        for feature_values in values:
+            yield tuple((val-mean)/max(sd, 0.01)
```
Comment (he7d3r) [inline revscores/scorers/linear_svc.py]: Is this division by 0.01 a good default when all the values are identicall (sd = 0)?

Comment (halfak) [inline revscores/scorers/linear_svc.py]: Well, in that case, all val-mean would be zero and zero divided by 0.01 is zero.  Generally, features that are all identical aren't good predictors.

Comment (he7d3r) [inline revscores/scorers/linear_svc.py]: Oh, off course! Looks good then.

Comment (halfak) [inline revscores/scorers/linear_svc.py]: Anything else you want me to take a look at?

**Comment by halfak** [comment]
Merge

**Comment by halfak** [comment]
FYI: Accidentally clicked the close button.  I reopened the pull request.

File: revscores/scorers/linear_svc.py (hunk 1 above)
Comment (he7d3r) [inline revscores/scorers/linear_svc.py]: It is ok for now.

**Pull request merged**